content/guides/mobile/ios/quick-start/3-configure-box-app.md

```diff
@@ -0,0 +1,82 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Configure a Box App
+
+To start making authenticated API calls to the Box API with the **Box iOS
+SDK**, an **Access Token** will be needed. The simplest way to generate a valid
+token is to generate a new **Box App** and manually generate a short lived
+developer token.
+
+The developer token is generated through the developer console UI and will be
+valid for one hour before having to be manually refreshed.
+
+## Set up a Box app
+
+<Grid columns='2'>
+  <Choose option='ios.app_type' value='create_new' color='blue'>
+    # Create a new Box app
+
+    Create and configure a new Box JWT application from which a developer
+    token may be generated
+  </Choose>
+
+  <Choose option='ios.app_type' value='use_own' color='red'>
+    # Use an existing app
+
+    Use one of your existing Box JWT applications
+  </Choose>
```

Comment:
Im not 100% sure if we really need to use a Choose here. We can just say "You need a Box App, here is a way to set one up yet if you don't have one already.". It's a bit overkill to have the actual choices. 

FYI in the Postman Quick Start we did as this, as it determined if we used our own credentials.

Update: added some comments further down